torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -1714,6 +1716,10 @@ def gen_reinterpret_call(writer, args):
 
     def codegen_device_copy(self, src, dst):
         if config.abi_compatible:
+            # aoti_torch_tensor_copy_ takes AtenTensorHandle as input,
+            # while stack-allocation results in ArrayRefTensor
+            # so add `dst` to the list to prevent stack-allocation for `dst` later
+            self.stack_allocation_blocklist.add(dst)
```

Comment:
I think just setting `self.allow_stack_allocation = False` here should work.

Reply:
SlimDSNN should not trigger this, so the PR shouldn't affect SlimDSNN's generated code.